src/api/app/views/webui/user/user_profile_redesign/_basic_info.html.haml

```diff
@@ -1,4 +1,14 @@
-.basic-info{ class: feature_enabled?(:user_profile_redesign) ? 'in-place-editing' : '' }
+.editing-form.d-none
+  = render partial: 'webui/user/user_profile_redesign/edit_account_form',
+           locals: { displayed_user: user, role_titles: role_titles, account_edit_link: account_edit_link }
+
+.basic-info
+  - if configuration.accounts_editable?(user)
+    .d-flex.flex-row-reverse
+      = link_to('javascript:void(0);', id: 'toggle-in-place-editing', class: 'nav-link', remote: true, title: 'Edit Your Account') do
+        %i.fas.mr-2.fa-user-edit
```

Comment:
```suggestion
        %i.fas.fa-user-edit
```

Reply:
Then, it would look like this
![Screenshot_2021-03-18 Profile of Admin](https://user-images.githubusercontent.com/2581944/111646102-a769b680-8801-11eb-9fc3-050da1ae6e1b.png)
: